src/main/java/pl/agh/cs/io/api/windows/WindowsListenerRunner.java

```diff
@@ -9,11 +13,10 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 
 public class WindowsListenerRunner {
-    private final static long POLLING_DELAY_MILLIS = 1000;
+    private final static long POLLING_DELAY_MILLIS = 1;
```

Comment:
Czy to oznacza że programy będą sprawdzane 1000 razy w ciągu sekundy? Jeżeli tak, nie jest to zbyt często?